client/src/components/Model.js

```diff
@@ -1,9 +1,12 @@
+import API from "api"
+import encodeQuery from "querystring/encode"
```

Comment:
Please move this import back to where it was (alphabetical order on imported module name).